.pipelines/cni/singletenancy/linux-cniv2-template.yaml

```diff
@@ -0,0 +1,183 @@
+                  cd ../../../..
+                  kubectl get po -owide -A
+                  sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true TEST_DROPGZ=${{ parameters.testDropgz }}
+              name: "overlaye2e"
+              displayName: "Overlay Integration"
+          - ${{ if contains(parameters.clusterType, 'swift') }}:
+            - task: AzureCLI@1
+              inputs:
+                azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+                scriptLocation: "inlineScript"
+                scriptType: "bash"
+                addSpnToEnvironment: true
+                inlineScript: |
+                  echo "Start Integration Tests on Swift Cluster"
+                  make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                  kubectl cluster-info
+                  kubectl get po -owide -A
+                  sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_DROPGZ_VERSION=$(dropgzVersion) INSTALL_CNS=true INSTALL_AZURE_VNET=true TEST_DROPGZ=${{ parameters.testDropgz }}
+              name: "swifte2e"
+              displayName: "Swift Integration"
+      - template: ../../npm/npm-cni-integration-test.yaml
+        parameters:
+          clusterName: ${{ parameters.clusterName }}-$(commitID)
+          os: ${{ parameters.os }}
+          sub: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+          tag: $(npmVersion)
+          dependsOn: integration
+          continueOnError: true
+      - job: deploy_pods
+        displayName: "Scale Test"
+        dependsOn: npm_k8se2e
+        steps:
+          - template: ../load-test-templates/pod-deployment-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              scaleup: ${LINUX_SCALEUP}
+              os: ${{ parameters.os }}
+              iterations: ${LINUX_ITERATIONS}
+              nodeCount: ${{ parameters.nodeCount }}
+          - template: ../load-test-templates/validate-state-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: ${{ parameters.os }}
+              cni: cniv2
+      - job: restart_nodes
+        displayName: "Restart Test"
+        dependsOn: deploy_pods
+        steps:
+          - template: ../load-test-templates/restart-node-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              nodeCount: ${{ parameters.nodeCount }}
+              scaleup: ${LINUX_SCALEUP}
+          - template: ../load-test-templates/validate-state-template.yaml
+            parameters:
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: ${{ parameters.os }}
+              cni: cniv2
+              restartCase: "true"
+      - job: restart_cns
```

Comment:
Would be nice to have template for this too ?

Reply:
Make sense to me!